src/volume_calc.cpp

```diff
@@ -300,12 +300,16 @@ vector<VolumeCalculation::Result> VolumeCalculation::execute() const
           if (i_material == MATERIAL_VOID)
             continue;
 
-          const auto& mat = model::materials[i_material];
-          for (int k = 0; k < mat->nuclide_.size(); ++k) {
-            // Accumulate nuclide density
-            int i_nuclide = mat->nuclide_[k];
-            atoms(i_nuclide, 0) += mat->atom_density_[k] * f;
-            atoms(i_nuclide, 1) += std::pow(mat->atom_density_[k], 2) * var_f;
+          // Guards against atom density calculation when no cross_section is
+          // presents
+          if (settings::run_mode == RunMode::VOLUME_NO_XS) {
```

Comment:
Looks like the conditional is inverted here (and further down as well). Should it be `!=` instead of `==`?

Reply:
Yes you are rigth.

I wanted to issue the PR, but didn't had time to correct it lately...